LeaderboardBackend.Jobs/Core/Options.cs

```diff
@@ -0,0 +1,77 @@
+using System.ComponentModel.DataAnnotations;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Models.Requests;
+using BCryptNet = BCrypt.Net.BCrypt;
+
+namespace LeaderboardBackend.Jobs.Core;
+
+internal static class Options
+{
+	public static string StringLine(string optionName, Func<string?, bool>? validator = null, string invalidMessage = "Invalid input.")
+	{
+		if (validator is null)
+		{
+			validator = s => !string.IsNullOrWhiteSpace(s);
+		}
+
+		string? input = null;
+		do
+		{
+			Console.Write($"{optionName}: ");
+			input = Console.ReadLine();
+			if (!validator(input))
+			{
+				Console.WriteLine(invalidMessage);
+			}
+		} while (!validator(input));
+
+		return input ?? "";
+	}
+
+	public static bool YesOrNo(string question)
+	{
+		Console.Write($"{question} (y to say yes): ");
```

Comment:
We could do the conventional `y/[N]` thing instead of `y to say yes`

Reply:
Makes the code more annoying to check explicitly for `y/n` and I figured it wasn't a big enough deal

Reply:
Oh nah cuz `[N]` implies replying with nothing means a rejection, and since we only check for ys, it works out. It's a nitpick, of course

Reply:
True, I didn't think of it that way. I'll do that.